docs/kafka/kafka-bin-scripts-kafka/README.adoc

```diff
@@ -228,7 +216,7 @@ The preceding example uses the SASL/PLAIN authentication mechanism with the cred
 >Third message
 ----
 
-. Keep the producer running to use later when you create a consumer.
+. Keep the producer running to use later, when you create a consumer.
```

Comment:
This sentence structure is a little clunky. Suggest 'Keep the producer running so you can use it later when you create a consumer.'

Reply:
Yes, agree, that's clearer.